vpc-classiclink/templates/default/aws.credentials.erb

```diff
@@ -0,0 +1,3 @@
+[default]
+aws_access_key_id = <%=node["vpc-classiclink"]["classiclink_iam_id"] %>
```

Comment:
Not sure what you think, but I would prefer using an instance policy so we don't have to manage these keys at all.